foxglove/api/api.go

```diff
@@ -526,6 +526,7 @@ func (e ExtensionResponse) String() string {
 
 type PendingImportsRequest struct {
 	RequestId       string `json:"requestId" form:"requestId,omitempty"`
+	Key		string `json:"key" form:"key,omitempty"`
```

Comment:
nitpick: could you adjust the spacing/columns here to align with other struct fields?